libs/libc/aio/lio_listio.c

```diff
@@ -507,7 +507,7 @@ static int lio_waitall(FAR struct aiocb * const *list, int nent)
 int lio_listio(int mode, FAR struct aiocb * const list[], int nent,
                FAR struct sigevent *sig)
 {
-  FAR struct aiocb *aiocbp;
+  FAR struct aiocb *aiocbp = NULL;
```

Comment:
According to the NuttX coding standard, `aiocbp` should be initialized separately from its definition.
So it is better to move the NULL attribution to line 523

Reply:
Could you point out the rule? There are many many places initialize the variables directly.

Reply:
Sorry, that was a misinterpretation of the C89 standard from myself. LGTM